bucket/windows-terminal.json

```diff
@@ -19,11 +20,21 @@
     "installer": {
         "script": [
             "$winVer = [Environment]::OSVersion.Version",
-            "if (($winver.Major -lt '10') -or ($winVer.Build -lt 18362)) { throw 'At least Windows 10 18362 is required.' }",
+            "if (($winver.Major -lt '10') -or ($winVer.Build -lt 18362)) { error 'At least Windows 10 19H1 (build 18362) is required.'; break }",
             "Get-ChildItem \"$dir\" '*.msix' | Select-Object -ExpandProperty Fullname | Expand-7zipArchive -DestinationPath \"$dir\" -Removal",
             "Get-ChildItem \"$dir\\ProfileIcons\" '*.png' | Rename-Item -NewName { $_.Name.Replace('%7B', '{').Replace('%7D', '}') }"
         ]
     },
+    "post_install": [
+        "'install-context', 'uninstall-context' | ForEach-Object {",
+        "    if (Test-Path \"$bucketsdir\\extras\\scripts\\windows-terminal\\$_.reg\") {",
+        "        $wtPath = \"$dir\\wt.exe\".Replace('\\', '\\\\')",
+        "        $content = (Get-Content \"$bucketsdir\\extras\\scripts\\windows-terminal\\$_.reg\").Replace('$wt', $wtPath)",
+        "        if ($global) { $content = $content.Replace('HKEY_CURRENT_USER', 'HKEY_LOCAL_MACHINE') }",
+        "        Set-Content \"$dir\\$_.reg\" $content -Encoding Ascii -Force",
+        "    }",
+        "}"
+    ],
```

Comment:
```suggestion
    ],
    "uninstaller": {
        "script": [
            "if ($cmd -eq 'uninstall') {",
            "    reg import \"$dir\\uninstall-context.reg\"",
            "}"
        ]
    },
```

Reply:
I think many users would **forget to unregister the context menu** before uninstalling, which can cause some trouble. This **uninstaller** script avoids the problem.

"uninstall-context" will only execute when the user runs `scoop uninstall ...`, but not `scoop update ...`

Reply:
This is a good idea. I was surprised [Sublime Text](https://github.com/ScoopInstaller/Extras/blob/master/bucket/sublime-text.json) and [Notepad++](https://github.com/ScoopInstaller/Extras/blob/master/bucket/notepadplusplus.json) didn’t implemented such a feature (I can’t see any `uninstaller` field on those manifests).

I will add it to the manifest soon.

Reply:
> I was surprised Sublime Text and Notepad++ didn’t implemented such a feature

Thanks for telling me. I will add this to Sublime Text and Notepad++ later.